src/python/pants/backend/python/target_types.py

```diff
@@ -249,7 +249,15 @@ class PythonBinary(Target):
 
 
 class PythonTestsSources(PythonSources):
-    default = ("test_*.py", "*_test.py", "tests.py", "conftest.py")
+    default = (
+        "test_*.py",
+        "*_test.py",
+        "tests.py",
+        "conftest.py",
+        "test_*.pyi",
+        "*_test.pyi",
+        "tests.pyi",
```

Comment:
I don't love this. But I think it's important to include `.pyi` in `python_tests` too. If a user for some reason writes type stubs for their tests, then we don't want it to show up in, say, `setup-py`. That is, we don't want the test type stubs defaulting to a `python_library`.

(Even though it's really weird to write a type stub file for your test)

Reply:
> I don't love this.

Is it just the number of lines to express it? Or you don't like how much it globs?

I might recommend putting the `.pyi` globs right after the `.py` globs, I think it *might* make it easier to read. But it's fine either way.

Reply:
I don't like how many globs we now have. But I do think that this is the correct behavior. Just clunky.